sdk/storage/storage-blob/src/BlobBatch.ts

```diff
@@ -211,69 +216,56 @@ export class BatchDeleteRequest extends BatchRequest {
       span.end();
     }
   }
-}
-
-/**
- * A BatchSetTierRequest represents a batch set tier request, which consists of one or more set tier operations.
- *
- * @export
- * @class BatchSetTierRequest
- * @extends {BatchRequest}
- */
-export class BatchSetTierRequest extends BatchRequest {
-  constructor() {
-    super();
-  }
 
   /**
-   * Add a set tier operation(subrequest) to set the tier on a blob.
-   * The operation is allowed on a page blob in a premium
-   * storage account and on a block blob in a blob storage account (locally redundant
-   * storage only). A premium page blob's tier determines the allowed size, IOPS,
-   * and bandwidth of the blob. A block blob's tier determines Hot/Cool/Archive
-   * storage type. This operation does not update the blob's ETag.
-   * See [set blob tier details](https://docs.microsoft.com/en-us/rest/api/storageservices/set-blob-tier).
-   * The operation(subrequest) will be authenticated and authorized
+   * The setBlobAccessTier operation sets the tier on a blob.
+   * The operation is allowed on block blobs in a blob storage or general purpose v2 account.
+   *
+   * A block blob's tier determines Hot/Cool/Archive storage type.
+   * This operation does not update the blob's ETag.
+   * For detailed information about block blob level tiering
+   * see [hot, cool, and archive access tiers](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-storage-tiers).
+   * The operation will be authenticated and authorized
    * with specified credential.See [blob batch authorization details](https://docs.microsoft.com/en-us/rest/api/storageservices/blob-batch#authorization).
    *
    * @param {string} url The url of the blob resource to delete.
    * @param {Credential} credential The credential to be used for authentication and authorization.
```

Comment:
`SharedKeyCredential | AnonymousCredential | TokenCredential` 

Please go through types for JSDoc parameters